src/theme/ThemeProvider.js

```diff
@@ -7,12 +7,18 @@ const theme = {
     secondaryBackground: '#1D1B24',
     foreground: '#4F4A59',
     warning: '#E03131',
-    primary: '#31E0E0', // #06C1C0 for hacker app but these look visually close
+    primary: '#31E0E0',
     default: '#BEBEBE', // default-state border color for most components used in the hacker app
     highlight: 'rgba(255, 255, 255, 0.6)',
     text: '#fff',
     link: '#fff',
     linkHover: '#31E0E0', //TODO
+    textInput: {
+      default: '#06C1C0',
+      selectHover: 'rgba(77, 232, 194, 0.2)',
+      selectClick: 'rgba(77, 232, 194, 0.5)',
+    },
+    buttonText: '#150F3B',
```

Comment:
Adding more styles according to design specs

<img width="595" alt="Screen Shot 2020-11-15 at 12 55 01 AM" src="https://user-images.githubusercontent.com/38872354/99180685-342e2780-26dd-11eb-8505-0e3ed132b1ce.png">
<img width="247" alt="Screen Shot 2020-11-15 at 12 55 52 AM" src="https://user-images.githubusercontent.com/38872354/99180689-5162f600-26dd-11eb-9ab6-78eed2115c20.png">